compiler/bindings/python/iree/compiler/tools/import_onnx/__main__.py

```diff
@@ -132,6 +139,29 @@ def parse_arguments(argv=None) -> argparse.Namespace:
         " Defaults to the directory of the input file.",
         type=Path,
     )
+    parser.add_argument(
+        "--opset-version",
+        help="Allows specification of a newer opset_version to update the model"
+        " to before importing to MLIR. This can sometime assist with shape inference.",
+        type=int,
+    )
+    parser.add_argument(
+        "--numel-threshold",
+        help="Minimum number of elements for an initializer to be externalized. Only has an effect if 'externalize-params' is true.",
+        type=int,
+        default=100,
+    )
```

Comment:
numel :O https://bulbapedia.bulbagarden.net/wiki/Numel_(Pok%C3%A9mon)

I'd spell out `num-elements-threshold` to be explicit... or build this into a more explicit API that allows users to control which specific parameters get externalized.
* https://iree.dev/guides/parameters/
* https://iree.dev/guides/ml-frameworks/pytorch/#using-external-parameters